File: testPrisma.js
const { PrismaClient } = require('@prisma/client');
const prisma = new PrismaClient();

async function main() {
  try {
    // Test fetching users
    const users = await prisma.user.findMany();
    console.log('Fetched users:', users);

    // You can add more CRUD operations here to test further
  } catch (error) {
    console.error('Error during testing:', error);
  } finally {
    await prisma.$disconnect();
  }
}

main();

File: app/test_prisma.py
from prisma import Prisma
import asyncio

async def test_prisma():
    prisma = Prisma()
    await prisma.connect()
    users = await prisma.user.find_many()
    print(users)
    await prisma.disconnect()

if __name__ == "__main__":
    asyncio.run(test_prisma())
